### ISSUE TITLE
Couldn't boot into Gaming Mode

### ISSUE THREAD

**Issue by irfanhakim-as** [ublue-os/bazzite#724]
### Describe the bug

- Starting the PC boots into desktop mode.
- Executing the "Return" shortcut from the desktop logs the user out and takes the user back into desktop mode.

### What did you expect to happen?

- The `bazzite-deck` all AMD PC should boot into Gaming mode by default.
- Executing the "Return" shortcut from the desktop should take the user into Gaming mode.
- First time encountering this issue in the past ~year of gaming/using Bazzite (deck) on this custom HTPC.

### Output of `rpm-ostree status`

```shell
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:e179d5c8b5f91d241d6ad748224fe7a8b7098147f411007e1a7f63a941b94e7c
                Timestamp: 2024-01-30T08:57:28Z
                Initramfs: '-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf' 

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:be54d1b13ee9bef835fd8d67c6e46c66da31fa73e059427c1ffa1ed18853a4b3
                Timestamp: 2024-01-29T05:55:05Z
                Initramfs: '-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf'
```

### Hardware

```
chihiro@haku:/var/home/chihiro/Desktop$ neofetch
                                               chihiro@haku 
          .................                    ------------ 
      .;;;;;;;;;;;;;;;;;;;;;;;.                Host: B550I AORUS PRO AX 
    ,;;;;;;;;     ;;;;;;;;;;;;;;;.             OCI: Bazzite (Steam Deck & HTPC Edition) x86_64 
   .;;;;;;;;;  ▲  ;;;;;;;;;;;;;;;;;;           OS: Fedora release 39 (Thirty Nine) x86_64 
   ';;;;;;;;;     ;;;;;;;;;;;;;;;;;:;          Kernel: 6.6.14-201.fsync.fc39.x86_64 
   ';;                   ;;   .;;;;;;;:        Uptime: 5 hours, 3 mins 
   ';;  ◄             ►  ;;      .;;;;;'       Packages: 2552 (rpm), 31 (steam), 77 (flatpak) 
   ';;                   ;;       .;;;;;       Shell: bash 5.2.26 
   ';;;;;;;;;     ;;;;;;;;;;;;,    ,;;;;.      Resolution: 1920x1080 @ 60.00Hz 
   ';;;;;;;;;  ▼  ;;;;;;;;;;;;.    ;;;;;,      DE: Plasma 5.27.10 [KF5 5.113.0] [Qt 5.15.12] (x11) 
   .;;;;;;;;;     ;;;;;;;;;;;'    ,;;;;;.      WM: KWin 
    ';;;;;;;;;;;;;;;;;;;;;;;.     ;;;;;,       Theme: Breeze [GTK2/3] 
    .;;;;;;;;     ;;;;;;;;.     .;;;;;;.       Icons: Mkos-Big-Sur-Night [KDE/Qt/GTK2/3] 
     .;;;;;;;;                 ;;;;;;;.        Terminal: konsole 
       ;;;;;;;;             .;;;;;;;;          CPU: AMD Ryzen 5 5600G (12) @ 3.5GHz 
        .;;;;;;;;;      ;;;;:;;;;;;.           GPU: AMD ATI Radeon Vega Series / Radeon Vega Mobile Series 
           ;;;;;;;;;;;;;;;;;;;;;;              Memory: 4.27 GiB / 30.69 GiB (13%) 
              .;;;;;;;;;;;;;;.                 Disk (/): 375G / 953G (40%) 

```

### Extra information or context

Don't recall doing anything in particular to have caused this - definitely have never installed any "external" packages or made any tweaks outside of whatever's done with Bazzite. May have updated (successfully) through Gaming mode, or through the command, `ujust update` and restarted prior to this issue, but cannot confirm this.

**Comment by KyleGospo** [comment]
Could you upload all of the `.gamescope-*` files in your home directory? That'll help identify what may be happening here.

**Comment by irfanhakim-as** [comment]
Sure! Here are 4 files that match that wildcard pattern in my home directory.

<details>

<summary>.gamescope-cmd.log</summary>

```
/usr/bin/gamescope --max-scale 2 --adaptive-sync -e --xwayland-count 2 -O *,eDP-1 --default-touch-mode 4 --hide-cursor-delay 3000 --fade-out-duration 200 --cursor-scale-height 720 -R /run/user/1000/gamescope.tzjrGAH/startup.socket -T /run/user/1000/gamescope.tzjrGAH/stats.pipe
```

</details>

<details>

<summary>.gamescope-cmd.log.old</summary>

```
/usr/bin/gamescope --max-scale 2 --adaptive-sync -e --xwayland-count 2 -O *,eDP-1 --default-touch-mode 4 --hide-cursor-delay 3000 --fade-out-duration 200 --cursor-scale-height 720 -R /run/user/1000/gamescope.4BiZKF6/startup.socket -T /run/user/1000/gamescope.4BiZKF6/stats.pipe
```

</details>

<details>

<summary>.gamescope-stdout.log</summary>

```
vblank: Using timerfd.
wlserver: [backend/headless/backend.c:67] Creating headless backend
wlserver: [libseat] [libseat/backend/seatd.c:64] Could not connect to socket /run/seatd.sock: No such file or directory
wlserver: [libseat] [libseat/libseat.c:76] Backend 'seatd' failed to open seat, skipping
wlserver: [libseat] [libseat/libseat.c:73] Seat opened with backend 'logind'
wlserver: [backend/session/session.c:109] Successfully loaded libseat session
ATTENTION: default value of option vk_xwayland_wait_ready overridden by environment.
ATTENTION: default value of option vk_xwayland_wait_ready overridden by environment.
ATTENTION: default value of option vk_xwayland_wait_ready overridden by environment.
ATTENTION: default value of option vk_xwayland_wait_ready overridden by environment.
drm: warning: picking an arbitrary DRM device
drm: Immediate flips are not supported by the KMS driver
drm: Immediate flips disabled from environment
drm: found new connector 'DP-1'
drm: Connector make XXX model Union TV
drm: [colorimetry]: EDID with colorimetry detected. Using it
drm: [colorimetry]: r 0.625000 0.339844
drm: [colorimetry]: g 0.280273 0.594727
drm: [colorimetry]: b 0.155273 0.070312
drm: [colorimetry]: w 0.283203 0.297852
drm: found new connector 'HDMI-A-2'
drm: found new connector 'HDMI-A-1'
drm: Connectors:
drm:   DP-1 (disconnected)
drm:   HDMI-A-2 (connected)
drm:   HDMI-A-1 (disconnected)
drm: selecting connector HDMI-A-2
drm: selecting mode 1920x1080@60Hz
vulkan: selecting physical device 'AMD Radeon Graphics (RADV RENOIR)': queue family 1 (general queue family 0)
vulkan: physical device supports DRM format modifiers
vulkan: supported DRM formats for sampling usage:
vulkan:   AR24 (0x34325241)
vulkan:   XR24 (0x34325258)
vulkan:   AB24 (0x34324241)
vulkan:   XB24 (0x34324258)
vulkan:   RG16 (0x36314752)
vulkan:   NV12 (0x3231564E)
vulkan:   AB4H (0x48344241)
vulkan:   XB4H (0x48344258)
vulkan:   AB48 (0x38344241)
vulkan:   XB48 (0x38344258)
vulkan:   AB30 (0x30334241)
vulkan:   XB30 (0x30334258)
vulkan:   AR30 (0x30335241)
vulkan:   XR30 (0x30335258)
wlserver: Running compositor on wayland display 'gamescope-0'
wlserver: [backend/headless/backend.c:17] Starting headless backend
wlserver: [backend/libinput/backend.c:88] Starting libinput backend
wlserver: [backend/libinput/events.c:70] Adding Power Button [0:1]
wlserver: [backend/libinput/events.c:70] Adding Video Bus [0:6]
wlserver: [backend/libinput/events.c:70] Adding Power Button [0:1]
wlserver: [backend/libinput/events.c:70] Adding Logitech Wireless Mouse PID:4022 [1133:16418]
wlserver: [backend/libinput/events.c:70] Adding Logitech Wireless Keyboard PID:4023 [1133:16419]
wlserver: [backend/libinput/backend.c:129] libinput successfully initialized
wlserver: [xwayland/server.c:108] Starting Xwayland on :0
wlserver: [xwayland/server.c:108] Starting Xwayland on :1
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Could not resolve keysym XF86CameraAccessEnable
> Warning:          Could not resolve keysym XF86CameraAccessDisable
> Warning:          Could not resolve keysym XF86CameraAccessToggle
> Warning:          Could not resolve keysym XF86NextElement
> Warning:          Could not resolve keysym XF86PreviousElement
> Warning:          Could not resolve keysym XF86AutopilotEngageToggle
> Warning:          Could not resolve keysym XF86MarkWaypoint
> Warning:          Could not resolve keysym XF86Sos
> Warning:          Could not resolve keysym XF86NavChart
> Warning:          Could not resolve keysym XF86FishingChart
> Warning:          Could not resolve keysym XF86SingleRangeRadar
> Warning:          Could not resolve keysym XF86DualRangeRadar
> Warning:          Could not resolve keysym XF86RadarOverlay
> Warning:          Could not resolve keysym XF86TraditionalSonar
> Warning:          Could not resolve keysym XF86ClearvuSonar
> Warning:          Could not resolve keysym XF86SidevuSonar
> Warning:          Could not resolve keysym XF86NavInfo
Errors from xkbcomp are not fatal to the X server
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Could not resolve keysym XF86CameraAccessEnable
> Warning:          Could not resolve keysym XF86CameraAccessDisable
> Warning:          Could not resolve keysym XF86CameraAccessToggle
> Warning:          Could not resolve keysym XF86NextElement
> Warning:          Could not resolve keysym XF86PreviousElement
> Warning:          Could not resolve keysym XF86AutopilotEngageToggle
> Warning:          Could not resolve keysym XF86MarkWaypoint
> Warning:          Could not resolve keysym XF86Sos
> Warning:          Could not resolve keysym XF86NavChart
> Warning:          Could not resolve keysym XF86FishingChart
> Warning:          Could not resolve keysym XF86SingleRangeRadar
> Warning:          Could not resolve keysym XF86DualRangeRadar
> Warning:          Could not resolve keysym XF86RadarOverlay
> Warning:          Could not resolve keysym XF86TraditionalSonar
> Warning:          Could not resolve keysym XF86ClearvuSonar
> Warning:          Could not resolve keysym XF86SidevuSonar
> Warning:          Could not resolve keysym XF86NavInfo
Errors from xkbcomp are not fatal to the X server
wlserver: [types/wlr_compositor.c:692] New wlr_surface 0x55adb6c9a620 (res 0x55adb6d7c020)
wlserver: [xwayland/server.c:273] Xserver is ready
wlserver: [types/wlr_compositor.c:692] New wlr_surface 0x55adb6c94490 (res 0x55adb6d75180)
wlserver: [xwayland/server.c:273] Xserver is ready
pipewire: stream state changed: connecting
pipewire: stream state changed: paused
pipewire: stream available on node ID: 93
xwm: Embedded, no cursor set. Using left_ptr by default.
xwm: Embedded, no cursor set. Using left_ptr by default.
drm: [josh edid] BASE Checksum valid? Y
drm: [josh edid] Wrote new edid to: /home/chihiro/.config/gamescope/edid.bin
wlserver: Updating mode for xwayland server #0: 1920x1080@60
pipewire: renegotiating stream params (size: 1920x1080)
wlserver: [types/wlr_compositor.c:692] New wlr_surface 0x55adb6931180 (res 0x55adb6d32a00)
xwm: Rejecting WM_CHANGE_STATE to ICONIC for window 0x400007
wlserver: [types/wlr_compositor.c:692] New wlr_surface 0x55adb6e1d5d0 (res 0x55adb6d3f8c0)
xwm: got the same buffer committed twice, ignoring.
The XKEYBOARD keymap compiler (xkbcomp) reports:
The XKEYBOARD keymap compiler (xkbcomp) reports:
> > Warning:          Warning:          Unsupported maximum keycode 708, clipping.
Unsupported maximum keycode 708, clipping.
> >                                     X11 cannot support keycodes above 255.
X11 cannot support keycodes above 255.
> > Warning:          Warning:          Could not resolve keysym XF86CameraAccessEnable
Could not resolve keysym XF86CameraAccessEnable
> > Warning:          Warning:          Could not resolve keysym XF86CameraAccessDisable
Could not resolve keysym XF86CameraAccessDisable
> > Warning:          Warning:          Could not resolve keysym XF86CameraAccessToggle
Could not resolve keysym XF86CameraAccessToggle
> > Warning:          Warning:          Could not resolve keysym XF86NextElement
Could not resolve keysym XF86NextElement
> > Warning:          Warning:          Could not resolve keysym XF86PreviousElement
Could not resolve keysym XF86PreviousElement
> > Warning:          Warning:          Could not resolve keysym XF86AutopilotEngageToggle
Could not resolve keysym XF86AutopilotEngageToggle
> > Warning:          Warning:          Could not resolve keysym XF86MarkWaypoint
Could not resolve keysym XF86MarkWaypoint
> > Warning:          Warning:          Could not resolve keysym XF86Sos
Could not resolve keysym XF86Sos
> > Warning:          Warning:          Could not resolve keysym XF86NavChart
Could not resolve keysym XF86NavChart
> > Warning:          Warning:          Could not resolve keysym XF86FishingChart
Could not resolve keysym XF86FishingChart
> > Warning:          Warning:          Could not resolve keysym XF86SingleRangeRadar
Could not resolve keysym XF86SingleRangeRadar
> > Warning:          Warning:          Could not resolve keysym XF86DualRangeRadar
Could not resolve keysym XF86DualRangeRadar
> > Warning:          Warning:          Could not resolve keysym XF86RadarOverlay
Could not resolve keysym XF86RadarOverlay
> > Warning:          Warning:          Could not resolve keysym XF86TraditionalSonar
Could not resolve keysym XF86TraditionalSonar
> > Warning:          Warning:          Could not resolve keysym XF86ClearvuSonar
Could not resolve keysym XF86ClearvuSonar
> > Warning:          Warning:          Could not resolve keysym XF86SidevuSonar
Could not resolve keysym XF86SidevuSonar
> > Warning:          Warning:          Could not resolve keysym XF86NavInfo
Could not resolve keysym XF86NavInfo
Errors from xkbcomp are not fatal to the X server
Errors from xkbcomp are not fatal to the X server
xwm: We failed our modeset and have no mode to fall back to! (Initial modeset failed?): Invalid argument
(EE) failed to read Wayland events: Broken pipe
(EE) failed to write to Xwayland fd: Broken pipe
```

</details>

<details>

<summary>.gamescope-stdout.log.old</summary>

```
vblank: Using timerfd.
wlserver: [backend/headless/backend.c:67] Creating headless backend
wlserver: [libseat] [libseat/backend/seatd.c:64] Could not connect to socket /run/seatd.sock: No such file or directory
wlserver: [libseat] [libseat/libseat.c:76] Backend 'seatd' failed to open seat, skipping
wlserver: [libseat] [libseat/libseat.c:73] Seat opened with backend 'logind'
wlserver: [backend/session/session.c:109] Successfully loaded libseat session
ATTENTION: default value of option vk_xwayland_wait_ready overridden by environment.
ATTENTION: default value of option vk_xwayland_wait_ready overridden by environment.
ATTENTION: default value of option vk_xwayland_wait_ready overridden by environment.
ATTENTION: default value of option vk_xwayland_wait_ready overridden by environment.
drm: warning: picking an arbitrary DRM device
drm: Immediate flips are not supported by the KMS driver
drm: Immediate flips disabled from environment
drm: found new connector 'DP-1'
drm: Connector make XXX model Union TV
drm: [colorimetry]: EDID with colorimetry detected. Using it
drm: [colorimetry]: r 0.625000 0.339844
drm: [colorimetry]: g 0.280273 0.594727
drm: [colorimetry]: b 0.155273 0.070312
drm: [colorimetry]: w 0.283203 0.297852
drm: found new connector 'HDMI-A-2'
drm: found new connector 'HDMI-A-1'
drm: Connectors:
drm:   DP-1 (disconnected)
drm:   HDMI-A-2 (connected)
drm:   HDMI-A-1 (disconnected)
drm: selecting connector HDMI-A-2
drm: selecting mode 1920x1080@60Hz
vulkan: selecting physical device 'AMD Radeon Graphics (RADV RENOIR)': queue family 1 (general queue family 0)
vulkan: physical device supports DRM format modifiers
vulkan: supported DRM formats for sampling usage:
vulkan:   AR24 (0x34325241)
vulkan:   XR24 (0x34325258)
vulkan:   AB24 (0x34324241)
vulkan:   XB24 (0x34324258)
vulkan:   RG16 (0x36314752)
vulkan:   NV12 (0x3231564E)
vulkan:   AB4H (0x48344241)
vulkan:   XB4H (0x48344258)
vulkan:   AB48 (0x38344241)
vulkan:   XB48 (0x38344258)
vulkan:   AB30 (0x30334241)
vulkan:   XB30 (0x30334258)
vulkan:   AR30 (0x30335241)
vulkan:   XR30 (0x30335258)
wlserver: Running compositor on wayland display 'gamescope-0'
wlserver: [backend/headless/backend.c:17] Starting headless backend
wlserver: [backend/libinput/backend.c:88] Starting libinput backend
wlserver: [libseat] [libseat/backend/logind.c:137] Could not take device: No such device
wlserver: [backend/session/session.c:321] Failed to open device: '/dev/input/event15': Resource temporarily unavailable
wlserver: [backend/libinput/events.c:70] Adding Power Button [0:1]
wlserver: [backend/libinput/events.c:70] Adding Video Bus [0:6]
wlserver: [backend/libinput/events.c:70] Adding Power Button [0:1]
wlserver: [backend/libinput/events.c:70] Adding Logitech Wireless Mouse PID:4022 [1133:16418]
wlserver: [backend/libinput/events.c:70] Adding Logitech Wireless Keyboard PID:4023 [1133:16419]
wlserver: [backend/libinput/backend.c:129] libinput successfully initialized
wlserver: [xwayland/server.c:108] Starting Xwayland on :0
wlserver: [xwayland/server.c:108] Starting Xwayland on :1
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Could not resolve keysym XF86CameraAccessEnable
> Warning:          Could not resolve keysym XF86CameraAccessDisable
> Warning:          Could not resolve keysym XF86CameraAccessToggle
> Warning:          Could not resolve keysym XF86NextElement
> Warning:          Could not resolve keysym XF86PreviousElement
> Warning:          Could not resolve keysym XF86AutopilotEngageToggle
> Warning:          Could not resolve keysym XF86MarkWaypoint
> Warning:          Could not resolve keysym XF86Sos
> Warning:          Could not resolve keysym XF86NavChart
> Warning:          Could not resolve keysym XF86FishingChart
> Warning:          Could not resolve keysym XF86SingleRangeRadar
> Warning:          Could not resolve keysym XF86DualRangeRadar
> Warning:          Could not resolve keysym XF86RadarOverlay
> Warning:          Could not resolve keysym XF86TraditionalSonar
> Warning:          Could not resolve keysym XF86ClearvuSonar
> Warning:          Could not resolve keysym XF86SidevuSonar
> Warning:          Could not resolve keysym XF86NavInfo
Errors from xkbcomp are not fatal to the X server
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Could not resolve keysym XF86CameraAccessEnable
> Warning:          Could not resolve keysym XF86CameraAccessDisable
> Warning:          Could not resolve keysym XF86CameraAccessToggle
> Warning:          Could not resolve keysym XF86NextElement
> Warning:          Could not resolve keysym XF86PreviousElement
> Warning:          Could not resolve keysym XF86AutopilotEngageToggle
> Warning:          Could not resolve keysym XF86MarkWaypoint
> Warning:          Could not resolve keysym XF86Sos
> Warning:          Could not resolve keysym XF86NavChart
> Warning:          Could not resolve keysym XF86FishingChart
> Warning:          Could not resolve keysym XF86SingleRangeRadar
> Warning:          Could not resolve keysym XF86DualRangeRadar
> Warning:          Could not resolve keysym XF86RadarOverlay
> Warning:          Could not resolve keysym XF86TraditionalSonar
> Warning:          Could not resolve keysym XF86ClearvuSonar
> Warning:          Could not resolve keysym XF86SidevuSonar
> Warning:          Could not resolve keysym XF86NavInfo
Errors from xkbcomp are not fatal to the X server
wlserver: [types/wlr_compositor.c:692] New wlr_surface 0x557dfbd4f890 (res 0x557dfbe2bf60)
wlserver: [xwayland/server.c:273] Xserver is ready
wlserver: [types/wlr_compositor.c:692] New wlr_surface 0x557dfbd4a8a0 (res 0x557dfbe251e0)
wlserver: [xwayland/server.c:273] Xserver is ready
pipewire: stream state changed: connecting
pipewire: stream state changed: paused
pipewire: stream available on node ID: 93
xwm: Embedded, no cursor set. Using left_ptr by default.
xwm: Embedded, no cursor set. Using left_ptr by default.
drm: [josh edid] BASE Checksum valid? Y
drm: [josh edid] Wrote new edid to: /home/chihiro/.config/gamescope/edid.bin
wlserver: Updating mode for xwayland server #0: 1920x1080@60
pipewire: renegotiating stream params (size: 1920x1080)
wlserver: [types/wlr_compositor.c:692] New wlr_surface 0x557dfb9e0ab0 (res 0x557dfbde1570)
xwm: Rejecting WM_CHANGE_STATE to ICONIC for window 0x400007
wlserver: [types/wlr_compositor.c:692] New wlr_surface 0x557dfbece390 (res 0x557dfbde5cd0)
xwm: got the same buffer committed twice, ignoring.
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 708, clipping.
>                   X11 cannot support keycodes above 255.
> Warning:          Could not resolve keysym XF86CameraAccessEnable
> Warning:          Could not resolve keysym XF86CameraAccessDisable
> Warning:          Could not resolve keysym XF86CameraAccessToggle
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Unsupported maximum keycode 708, clipping.
>                   X11 cannot support keycodes above 255.
> Warning:          Could not resolve keysym XF86NextElement
> Warning:          Could not resolve keysym XF86PreviousElement
> Warning:          Could not resolve keysym XF86AutopilotEngageToggle
> Warning:          Could not resolve keysym XF86MarkWaypoint
> Warning:          Could not resolve keysym XF86Sos
> Warning:          Could not resolve keysym XF86NavChart
> Warning:          Could not resolve keysym XF86FishingChart
> Warning:          Could not resolve keysym XF86SingleRangeRadar
> Warning:          Could not resolve keysym XF86DualRangeRadar
> Warning:          Could not resolve keysym XF86RadarOverlay
> Warning:          Could not resolve keysym XF86TraditionalSonar
> Warning:          Could not resolve keysym XF86ClearvuSonar
> Warning:          Could not resolve keysym XF86SidevuSonar
> Warning:          Could not resolve keysym XF86NavInfo
Errors from xkbcomp are not fatal to the X server
> Warning:          Could not resolve keysym XF86CameraAccessEnable
> Warning:          Could not resolve keysym XF86CameraAccessDisable
> Warning:          Could not resolve keysym XF86CameraAccessToggle
> Warning:          Could not resolve keysym XF86NextElement
> Warning:          Could not resolve keysym XF86PreviousElement
> Warning:          Could not resolve keysym XF86AutopilotEngageToggle
> Warning:          Could not resolve keysym XF86MarkWaypoint
> Warning:          Could not resolve keysym XF86Sos
> Warning:          Could not resolve keysym XF86NavChart
> Warning:          Could not resolve keysym XF86FishingChart
> Warning:          Could not resolve keysym XF86SingleRangeRadar
> Warning:          Could not resolve keysym XF86DualRangeRadar
> Warning:          Could not resolve keysym XF86RadarOverlay
> Warning:          Could not resolve keysym XF86TraditionalSonar
> Warning:          Could not resolve keysym XF86ClearvuSonar
> Warning:          Could not resolve keysym XF86SidevuSonar
> Warning:          Could not resolve keysym XF86NavInfo
Errors from xkbcomp are not fatal to the X server
xwm: We failed our modeset and have no mode to fall back to! (Initial modeset failed?): Invalid argument
(EE) failed to read Wayland events: Broken pipe
(EE) failed to read Wayland events: Connection reset by peer
```

</details>

**Comment by irfanhakim-as** [comment]
Another thing that may or may not be of note, is that when Bazzite made the change to default to Wayland, I toggled it back to X11 using the command provided by Bazzite, `ujust _toggle_wayland`.

**Comment by KyleGospo** [comment]
That won't affect anything here, gamemode is always using a Wayland compositor (gamescope).

I'm pushing a new update now that updates Gamescope, Mutter, and a couple Wayland packages. Might be worth a re-test once it's built.

**Comment by irfanhakim-as** [comment]
> That won't affect anything here, gamemode is always using a Wayland compositor (gamescope).

Yea, that's what I thought too.

> I'm pushing a new update now that updates Gamescope, Mutter, and a couple Wayland packages. Might be worth a re-test once it's built.

Thanks for the update! Please let me know when I'm able to test it.

**Comment by irfanhakim-as** [comment]
I've performed an update:

```sh
ujust update
```

and restarted, and it still gets me to Desktop mode. I then tried to manually get into Gaming mode by using the `Return` shortcut, and it still reloaded me back into Desktop mode.

My current `rpm-ostree status`:

```
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:3899a7b5bacd5419656c2008c938b93d2789499939b7a49874b52cff9fb35876
                Timestamp: 2024-01-31T05:45:08Z
                Initramfs: '-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf' 

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:e179d5c8b5f91d241d6ad748224fe7a8b7098147f411007e1a7f63a941b94e7c
                Timestamp: 2024-01-30T08:57:28Z
                Initramfs: '-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf'
```

**Comment by KyleGospo** [comment]
Very strange, one other thing to try -- can you open Steam from desktop and switch to the beta branch in settings, then restart it and let it update before booting back to gamemode?

**Comment by irfanhakim-as** [comment]
Ok, from Steam desktop, I navigated into `Settings > Interface` and updated the `Client Beta Participation` setting from `Steam Deck Stable` to `Steam Deck Beta`.

It then prompted me to restart Steam, and after it does, it proceeded to update Steam before it launches the Steam interface.

I closed the Steam interface, then executed the `Return` shortcut, and it just brought me back into Desktop mode once again.

I also tried restarting the PC, and that also led me back into Desktop mode.

**Comment by EyeCantCU** [comment]
Thank you for the report. Can you get me -
`systemctl status bazzite-autologin`

And -
`systemctl status sddm`

**Comment by irfanhakim-as** [comment]
Sure!

`systemctl status bazzite-autologin`:

```
○ bazzite-autologin.service - Enables Bazzite autologin
     Loaded: loaded (/usr/lib/systemd/system/bazzite-autologin.service; enabled; preset: disabled)
    Drop-In: /usr/lib/systemd/system/service.d
             └─10-timeout-abort.conf
     Active: inactive (dead) since Wed 2024-01-31 14:56:49 +08; 5h 31min ago
    Process: 1099 ExecStart=/usr/bin/bazzite-autologin (code=exited, status=0/SUCCESS)
   Main PID: 1099 (code=exited, status=0/SUCCESS)
        CPU: 42ms

Jan 31 14:56:49 haku systemd[1]: Starting bazzite-autologin.service - Enables Bazzite autologin...
Jan 31 14:56:49 haku systemd[1]: bazzite-autologin.service: Deactivated successfully.
Jan 31 14:56:49 haku systemd[1]: Finished bazzite-autologin.service - Enables Bazzite autologin.
```

`systemctl status sddm`:

```
● sddm.service - Simple Desktop Display Manager
     Loaded: loaded (/usr/lib/systemd/system/sddm.service; enabled; preset: enabled)
    Drop-In: /usr/lib/systemd/system/service.d
             └─10-timeout-abort.conf
     Active: active (running) since Wed 2024-01-31 14:58:07 +08; 5h 29min ago
       Docs: man:sddm(1)
             man:sddm.conf(5)
   Main PID: 5739 (sddm)
      Tasks: 2 (limit: 37608)
     Memory: 6.9M
        CPU: 39ms
     CGroup: /system.slice/sddm.service
             └─5739 /usr/bin/sddm

Jan 31 14:58:07 haku systemd[1]: Started sddm.service - Simple Desktop Display Manager.
```

**Comment by KyleGospo** [comment]
I may have found a regression in the Kernel, let me get that patched up and I'll message you when it's ready

**Comment by irfanhakim-as** [comment]
Thanks for the update!

**Comment by KyleGospo** [comment]
Try the latest build, this may be fixed. Issue was an upstream change to xwayland for security reason that unfortunately broke gamescope for some users.

**Comment by irfanhakim-as** [comment]
I've updated again with `ujust update` and it's still not fixed unfortunately, for me :(

My current version:

```
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:d8f1dc46a24ae3100bf3ccb52ff63727d119906436c3f4980bad16a302e1d886
                Timestamp: 2024-02-01T07:36:27Z
                Initramfs: '-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf' 

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:3899a7b5bacd5419656c2008c938b93d2789499939b7a49874b52cff9fb35876
                Timestamp: 2024-01-31T05:45:08Z
                Initramfs: '-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf'
```

I'm also still in the `Steam Deck Client Beta` channel from previous test.

**Comment by irfanhakim-as** [comment]
I just remembered that I've to note that in a previous update, before gamemode broke (though can't guarantee if it broke exactly after doing these, I doubt it), I also ran these commands besides `ujust update`/`just update`:

```sh
  ujust fix-steam-download-speed
  ujust restore-bazzite-breeze-gtk-theme
  ujust restore-original-terminal
  ujust get-simpledeckytdp
```

Please let me know if there's anything else I could try/share to further debug this issue.

To start, here's `sudo dmesg -T` after executing the `Return` shortcut, and being taken back to Desktop mode again:

<details>

<summary>dmesg</summary>

```
[Fri Feb  2 03:05:31 2024] evdi: [I] (card3) Closed by Task 8385 (Xorg) of process 8385 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card2) Closed by Task 8385 (Xorg) of process 8385 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card1) Closed by Task 8385 (Xorg) of process 8385 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card0) Closed by Task 8385 (Xorg) of process 8385 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card0) Opened by Task 13417 (Xorg.wrap) of process 13417 (Xorg.wrap)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card0) Closed by Task 13417 (Xorg.wrap) of process 13417 (Xorg.wrap)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card1) Opened by Task 13417 (Xorg.wrap) of process 13417 (Xorg.wrap)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card1) Closed by Task 13417 (Xorg.wrap) of process 13417 (Xorg.wrap)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card2) Opened by Task 13417 (Xorg.wrap) of process 13417 (Xorg.wrap)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card2) Closed by Task 13417 (Xorg.wrap) of process 13417 (Xorg.wrap)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card3) Opened by Task 13417 (Xorg.wrap) of process 13417 (Xorg.wrap)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card3) Closed by Task 13417 (Xorg.wrap) of process 13417 (Xorg.wrap)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card0) Opened by Task 1226 (systemd-logind) of process 1226 (systemd-logind)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card1) Opened by Task 1226 (systemd-logind) of process 1226 (systemd-logind)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card2) Opened by Task 1226 (systemd-logind) of process 1226 (systemd-logind)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card3) Opened by Task 1226 (systemd-logind) of process 1226 (systemd-logind)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card0) Opened by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card0) Closed by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card1) Opened by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card1) Closed by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card2) Opened by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card2) Closed by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card3) Opened by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card3) Closed by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card0) Opened by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:31 2024] evdi: [I] (card0) Closed by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:32 2024] evdi: [I] (card3) Closed by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:32 2024] evdi: [I] (card2) Closed by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:32 2024] evdi: [I] (card1) Closed by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:32 2024] evdi: [I] (card0) Closed by Task 13417 (Xorg) of process 13417 (Xorg)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card0) Opened by Task 13814 (Xorg.wrap) of process 13814 (Xorg.wrap)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card0) Closed by Task 13814 (Xorg.wrap) of process 13814 (Xorg.wrap)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card1) Opened by Task 13814 (Xorg.wrap) of process 13814 (Xorg.wrap)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card1) Closed by Task 13814 (Xorg.wrap) of process 13814 (Xorg.wrap)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card2) Opened by Task 13814 (Xorg.wrap) of process 13814 (Xorg.wrap)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card2) Closed by Task 13814 (Xorg.wrap) of process 13814 (Xorg.wrap)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card3) Opened by Task 13814 (Xorg.wrap) of process 13814 (Xorg.wrap)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card3) Closed by Task 13814 (Xorg.wrap) of process 13814 (Xorg.wrap)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card0) Opened by Task 1226 (systemd-logind) of process 1226 (systemd-logind)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card1) Opened by Task 1226 (systemd-logind) of process 1226 (systemd-logind)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card2) Opened by Task 1226 (systemd-logind) of process 1226 (systemd-logind)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card3) Opened by Task 1226 (systemd-logind) of process 1226 (systemd-logind)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card0) Opened by Task 13814 (Xorg) of process 13814 (Xorg)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card0) Closed by Task 13814 (Xorg) of process 13814 (Xorg)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card1) Opened by Task 13814 (Xorg) of process 13814 (Xorg)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card1) Closed by Task 13814 (Xorg) of process 13814 (Xorg)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card2) Opened by Task 13814 (Xorg) of process 13814 (Xorg)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card2) Closed by Task 13814 (Xorg) of process 13814 (Xorg)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card3) Opened by Task 13814 (Xorg) of process 13814 (Xorg)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card3) Closed by Task 13814 (Xorg) of process 13814 (Xorg)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card0) Opened by Task 13814 (Xorg) of process 13814 (Xorg)
[Fri Feb  2 03:05:44 2024] evdi: [I] (card0) Closed by Task 13814 (Xorg) of process 13814 (Xorg)
```

</details>

**Comment by ParasiticSquid** [comment]
Im also running into a issue where gamescope simply doesn't work. Had to enable desktop autologin since otherwise game mode black screens until you use a tty to login and run "steamos-session-select plasma". That's on top of steam not saving my login no matter what I do but that's another issue entirely.

My card is a radeon r7 240 since the computer using the bazzite install is my old one from a few years ago. Decided to try and convert it into a living room game console of sorts. But game mode not working kind of put a stop to those plans atm. Thing is with all research ive done and my own experience with this card in the past it SHOULD work. This card has vulkan support and the amdgpu driver has support for it yet gamescope just doesn't do anything no matter what I do. Constantly complains about not being able to start vulkan basically. Been at it for 3 days to no effect oof

As for irfanhakim-as, if they log in directly to the desktop bypassing game mode entirely maybe they have the desktop auto login toggle enabled? I used "ujust toggle-autologin" to bypass game mode until a miracle comes along or I find out why gamescope and steam are being difficult and it boots me directly to desktop instead of having to deal with the black screen each boot.

**Comment by irfanhakim-as** [comment]
I've updated again, even went through the Bazzite Portal wizard, shut down and power cycled the PC - and still the same result. Such a shame since I've just only fixed an unrelated hardware issue on my HTPC recently and was finally able to game on it with Bazzite, but now I'm once again left with a HTPC I could not game on.

```
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:d7f7f74a7f42c1bdba8638f36d668f19ba28f3b066b58df54f99c23eb135609e
                Timestamp: 2024-02-02T02:33:18Z
                Initramfs: '-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf' 

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:d8f1dc46a24ae3100bf3ccb52ff63727d119906436c3f4980bad16a302e1d886
                Timestamp: 2024-02-01T07:36:27Z
                Initramfs: '-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf'
```

Thanks for keeping the report up, I'll continue to update from time to time and hope gamemode will _magically_ work again.

**Comment by RandomLegend** [comment]
i have the same issue after updating yesterday.

Clicking on return to gamemode shuts down steam, turns the display black for 1 sec and throws me back into desktop mode and restarts steam.

Also tried ujust toggle-autologin

**Comment by djranm** [comment]
I'm also experiencing this issue, as well as Steam in desktop mode not saving my credentials no matter what, on a brand new install of 2.2.0 which I then updated via topgrade. The two issues might be related?

Next step I was going to try is reverting to 2.2.0.

**Comment by ParasiticSquid** [comment]
In my case I started from a fedora kinoite install and rebased to a bazzite-deck image due to the computer I tried installing on not booting the bazzite USB and ventoy booting with the bazzite install stuff still not working but at least trying initially at first. I don't even know why, the bazzite USB just wouldn't boot no matter what yet it did just fine on every other computer I own when testing the USB on those but heard rebasing from fedora kinoite or universal blue( which had the same USB issue on the computer ) were basically the same thing as a full bazzite install and tried that.

Maybe that's a possible issue? I googled it for hours on end but no one else has has similar issues from a rebase to my knowledge and %90 of similar issues I found with gamescope and game mode not working seem to have to do mostly with nvidia cards. I don't have a nvidia card and I know for fact vulkan is functional on the system since I can even do vkcube on both wayland and x as well as vulkaninfo. Not to mention waydrdoid works pefectly find so wayland isn't the issue.

I got so fed up with everything I basically gave up and installed batocera instead. Maybe when I actually know why things aren't working ill revisit bazzite but for now.. it's been weeks and it's just too much.

**Comment by irfanhakim-as** [comment]
I've been updating from time to time and testing it out but stopped doing so for a while since there's been no updates on this issue, and the issue remained to exist.

Today though I've updated again and it after it restarted, it booted right back into Gaming mode (as intended). I then tested further by going to Desktop mode and back to Gaming mode again through the shortcut, I also turned off my HTPC entirely and powered it back on again. In all cases, the PC went into Gaming mode perfectly.

```
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:3f5b838c5a96ab04972ced136c4c1602f5d68f58d900836d6ede4ad426ec8714
                  Version: 39.20240301.0 (2024-03-02T00:21:09Z)
                Initramfs: '-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf' 

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:3f5b838c5a96ab04972ced136c4c1602f5d68f58d900836d6ede4ad426ec8714
                  Version: 39.20240301.0 (2024-03-02T00:21:09Z)
                Initramfs: '-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf'
```

I'm happy to be able to use my HTPC (on Gaming mode) again - can others (with this issue) verify if this is finally resolved by updating and testing it out? You can update from Desktop mode through the terminal (Konsole) with this command:

```sh
ujust update
```

Thanks!

**Comment by irfanhakim-as** [comment]
@bgiesing We're on a different update channel, it'd be more helpful if you were to add in your `rpm-ostree status` output to get the exact version you're running.

In my case, and the version I'm on which I've shared, this issue of booting and getting into Gaming Mode is solved - but now I'm getting a new, more annoying issue that undoubtedly came into existence only after the update:

I often have my HTPC running 24/7 - only setting it (via Plasma Settings) to sleep after some time of inactivity in Desktop mode. Usually, in the past, things work as expected where come night time when I want to use it I'd just wake it up via my peripherals i.e. mouse. After this specific update though, not always but very often to have it happen to me many times now - my PC just wouldn't wake. Not via peripherals, and not even via the power button (single press). My PC's RGB on the motherboard or RAM would remain lit (which it normally does even when asleep since it's powered), but the LED on the power button would not light up as it normally would when it wakes/awake, and my display isn't getting any signal either.

The only "fix" then is to force a shutdown by long pressing the power button, and booting it back up. This honestly is really weird behaviour I haven't encountered before in the past with this PC on Bazzite or any other distro - but it does continue the curse I have of not being able to use this "exclusively" Bazzite HTPC of mine, encountering one serious issue after another.

**Comment by Lazorne** [comment]
[lazlogs.tar.gz](https://github.com/ublue-os/bazzite/files/14667848/lazlogs.tar.gz)

I honestly have never gotten gamemode to work on my AMD :) 

Here are all the logs, a wild guess could it have something to do with Dual Monitor setups?

**Comment by alexcash** [comment]
This happened to me when I had no password set up for my user, which the installer lets you do. I was able to run ‘passwd <username>’ in terminal and reboot. That fixed the issue for me.

**Comment by devjaizon** [comment]
Mine is the same. I already installed twice and it just won't launch gamemode. Also, Steam credentials aren't being saved, do I have to login every time i boot my PC. I'm Running bazzite-deck-stable

**Comment by ScrambledLogic** [comment]
I"m also having this issue, as well as Steam refusing to remember my login credentials, when installing with the bazzite-deck-stable iso. I also tried installing with the bazzite-stable iso and rebasing to bazzite-deck-stable; before rebasing, Steam will remember my login, but after rebasing it will continuously forget again. Gaming Mode refused to work in either case.

**Comment by Stormer1911** [comment]
Exact same issue with the exact same GPU

**Comment by mmyllari** [comment]
Same problems, gamemode does not start, steam login does not get stored. GPU is RX 480. Please prioritize this issue 👍

**Comment by rccarlson** [comment]
Having this same issue...

**Comment by reisenmiao** [comment]
I had similar symptoms, but my case was on an NVIDIA GPU, so it may not have the same root cause as the original AMD issue.

### Symptoms

Before switching drivers, I had the following problems:

* I could not switch to Game Mode.
* There was no sound through DisplayPort.
* HDMI audio worked normally.

### System state before the fix

My system was using the `nouveau` driver for the NVIDIA GPU.

### Fix

Rebasing to the NVIDIA Bazzite image fixed the issue for me:

```bash
sudo rpm-ostree rebase ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck-nvidia:stable
systemctl reboot
```

### Result

After rebooting, the system used the NVIDIA driver instead of `nouveau`.

Both issues were fixed:

* Game Mode worked.
* DisplayPort audio worked.
